iChef-WebUi/angular.json

```diff
@@ -89,8 +89,12 @@
                     "configurations": {
                         "production": {
                             "browserTarget": "iChef-WebUi:build:production"
+                        },
```

Comment:
I think you need to remove changes from this file

Reply:
+1